more/free-programming-playgrounds.md

```diff
@@ -53,6 +54,12 @@
 * [Vim](#vim)
 
 
+### Algorithms
+
+* [The Algorithm Visualizer](https://algorithm-visualizer.org)
+* [The Algorithms](https://the-algorithms.com)
```

Comment:
this one isn't a playground. I think it fits better in books.

Reply:
The one you are talking about is the algorithms (https://the-algorithms.com)? I can move it to books. :)

Reply:
yes please

Reply:
```suggestion
```
moved to books in #10986